Classes/GlobalStateExplorers/KeyChain/FLEXKeychain.h

```diff
@@ -0,0 +1,198 @@
+//
+//  SSKeychain.h
+//  SSKeychain
+//
+//  Created by Sam Soffes on 5/19/10.
+//  Copyright (c) 2010-2014 Sam Soffes. All rights reserved.
+//
+
+#if __has_feature(modules)
+	@import Foundation;
+#else
+	#import <Foundation/Foundation.h>
+#endif
+
+/**
+ Error code specific to SSKeychain that can be returned in NSError objects.
+ For codes returned by the operating system, refer to SecBase.h for your
+ platform.
+ */
+typedef NS_ENUM(OSStatus, SSKeychainErrorCode) {
```

Comment:
What does `SS` means here? Prefer to use `FLEXKeychain` as prefix

Reply:
it is from [SSKeyChain](https://github.com/soffes/SAMKeychain)

Reply:
@matrush Fixed, testing the UI then I'll push my changes and merge